server/datastore/mysql/apple_mdm.go

```diff
@@ -3405,41 +3429,41 @@ DELETE FROM
 WHERE
   team_id = ? AND %s
 `
-	andIdentNotInList := "identifier NOT IN (?)" // added to fmtDeleteStmt if needed
+	andNameNotInList := "name NOT IN (?)" // added to fmtDeleteStmt if needed
 
 	const loadExistingDecls = `
 SELECT
-  identifier,
+  name,
   declaration_uuid,
   raw_json
 FROM
   mdm_apple_declarations
 WHERE
   team_id = ? AND
-  identifier IN (?)
+  name IN (?)
 `
 
 	var declTeamID uint
 	if tmID != nil {
 		declTeamID = *tmID
 	}
 
-	// build a list of identifiers for the incoming declarations, will keep the
+	// build a list of names for the incoming declarations, will keep the
```

Comment:
I switched the logic to use names instead of identifiers as reference to identify what changed - this is because we have the macOS updates DDM that is by name. I don't _think_ it changes things much, both name and identifier must be unique paired with the team_id, but just a heads-up in case you see a potential issue with this change.

Reply:
SGTM 👍